src/code_systems/panelappCodes.ts

```diff
@@ -0,0 +1,114 @@
+import { RequiredCoding } from "./types";
+
+export const diseases: RequiredCoding[] = [
+  {
+    system: "http://panelapp.genomicsengland.co.uk/api/v1/entities",
+    code: "R14",
```

Comment:
Yeah its a bit frustrating, we're aligning with what the redcap form is using, so that both ways of saving data are consistent, though snomed-CT can be more specific